src/deepEqual.js

```diff
@@ -21,6 +21,32 @@
  */
 function deepEqual(a, b) {
   // write code here
+  let typeOfA = typeof a;
+  let typeOfB = typeof b;
+
+  if (typeOfA === typeOfB && a === b) { // Если сущности равны сами по себе...
```

Comment:
1. don't write comments in Russian. only in English. 
2. comment should be before the line. 

// comment goes here.
if (typeOfA === typeOfB && a === b) {
 ...
} else if () {
// comment goes here. 
... 
}

Reply:
Ok, understood. I'll check it.
P. S. Thanks for spearing no affort to coach us!